ingest/src/main/java/com/microsoft/azure/kusto/ingest/ResourceManager.java

```diff
@@ -38,60 +41,33 @@
 import java.util.concurrent.locks.ReadWriteLock;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
 
-class ResourceManager implements Closeable {
+public class ResourceManager implements Closeable, IngestionResourceManager {
+    public static final String SERVICE_TYPE_COLUMN_NAME = "ServiceType";
+    private static final long REFRESH_INGESTION_RESOURCES_PERIOD = 1000L * 60 * 60; // 1 hour
```

Comment:
All of these would IMHO be more readable as:
`private static final Long REFRESH_INGESTION_RESOURCES_PERIOD_MILLIS = TimeUnit.HOURS.toMillis(1);`

Reply:
I think this api is ugly but it's probably better.